sunpy/map/mapcube.py

```diff
@@ -232,19 +248,28 @@ def peek(self, gamma=None, resample=None, **kwargs):
         Examples
         --------
         >>> cube = sunpy.Map(files, cube=True)
-        >>> ani = cube.plot(colorbar=True)
+        >>> ani = cube.peek(colorbar=True)
         >>> plt.show()
 
         Plot the map at 1/2 original resolution
 
         >>> cube = sunpy.Map(files, cube=True)
-        >>> ani = cube.plot(resample=[0.5, 0.5], colorbar=True)
+        >>> ani = cube.peek(resample=[0.5, 0.5], colorbar=True)
+        >>> plt.show()
+
+        Plot the map with the limb at each time step
+
+        >>> def myplot(fig, ax, map):
+        ...    p = map.draw_limb()
+        ...    return p
+        >>> cube = sunpy.Map(files, cube=True)
+        >>> ani = cube.peek(plot_function=myplot)
```

Comment:
The idea of `plot()` is that it returns a matplotlib animator object, so that it can be saved to a file. The peek() method is for interactive work.

I will add an example to `plot()`.